custom/icds_reports/models/helper.py

```diff
@@ -11,3 +17,22 @@ class IcdsMonths(models.Model):
     class Meta:
         managed = False
         db_table = 'icds_months'
+
+
+class IcdsFile(models.Model):
+    blob_id = models.CharField(max_length=255)
+    data_type = models.CharField(max_length=255)
+    file_added = models.DateField(auto_now=True)
+
+    def store_file_in_blobdb(self, file, filename):
+        blob_db = get_blob_db()
+        blob_db.put(file, filename, bucket=ICDS_BUCKET, timeout=EXPIRED)
+        self.save()
```

Comment:
I think this method should be changed to:
```py
def store_file_in_blobdb(self, file):
    blob_db = get_blob_db()
    blob_db.put(file, self.blob_id, bucket=ICDS_BUCKET, timeout=EXPIRED)
```
Use `self.blob_id` rather than the passed in `filename` since `filename` has already been saved in `self.blob_id` by `IcdsFile.objects.get_or_create(blob_id=filename, ...`, and `self.blob_db` is not being updated here. `self.save()` is not necessary because nothing about this object has changed.

Also, reusing blob identifiers can result in race conditions when retrieving blob content. Is there any concern that the cached content could be outdated?